docs/src/main/paradox/scala/http/compatibility-guidelines.md

```diff
@@ -71,3 +71,39 @@ Maven
     </dependency>
     ```
     @@@
+    
+The same issue using `akka-http-testkit` library. So in this case, it is necessary to add:
```

Comment:
Maybe we could just add this information to the above snippet and just add comments that you only need to add if you want to add the testkit. WDYT?

Reply:
Agee. I will do it today.